l10n_it_ricevute_bancarie/tests/test_invoice_due_cost.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+#
+# Author: Andrea Gallina
+# ©  2015 Apulia Software srl
+#
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp.tests.common import TransactionCase
+
+
+class TestInvoiceDueCost(TransactionCase):
+
+    def _create_pterm(self):
+        return self.env['account.payment.term'].create({
+            'name': 'Ri.Ba. 30/60',
+            'riba': True,
+            'payment_cost': 5.00,
+            'line_ids': [
+                (0, 0,
+                 {'value': 'procent', 'days': 30,
+                  'days2': -1, 'value_amount': 0.50}),
+                (0, 0,
+                 {'value': 'balance', 'days': 60,
+                  'days2': -1})
+            ]
+        })
+
+    def _create_pterm2(self):
+        return self.env['account.payment.term'].create({
+            'name': 'Ri.Ba. 30',
+            'riba': True,
+            'payment_cost': 5.00,
+            'line_ids': [
+                (0, 0,
+                 {'value': 'balance', 'days': 30,
+                  'days2': -1})
+            ]
+        })
+
+    def _account_expense(self):
+        return self.env['account.account'].create({
```

Comment:
It's possible to obtain values to create from another function? So, other modules can extend dict informaton